pkg/kapp/resources/resources.go

```diff
@@ -169,6 +169,24 @@ func (c *ResourcesImpl) All(resTypes []ResourceType, opts AllOpts) ([]Resource,
 				}
 			}
 
+			// In case we successfully get the list for limited scope, we will
+			// try to run namespace scoped lists for rest of the allowed namespaced
+			if resType.Namespaced() && nsScopeLimited {
+				for i := 1; i < len(c.opts.FallbackAllowedNamespaces); i++ {
```

Comment:
Lets be cool kids and use `range` here?

Reply:
range would start from 0 again :(
We are not kids anymore xD xD

Reply:
Oh I see 🤣 
We are dodging `""`, alrighty makes sense

Reply:
Well, not `""`, but the first namespace that is there in fallbackAllowedNamespaces.